src/pkg/validation/validationError.go

```diff
@@ -11,7 +11,7 @@ import (
 // if the failed value is a map or slice, it will be omitted
 type ValidatorError struct {
 	jsonschema.BasicError
```

Comment:
More of a question outside scope of this PR - why are the keys in the yaml output all lowercase vs camel case of the JSON output? 

some nuance to the marshall process where this natively becomes camel vs unhandled with yaml?

Reply:
Looks like this is because I am "embedding/extending" the `BasicError` struct from the jsonschema library and I just noticed that they do not have yaml tags in their definitions, so I may need to create our own or open a PR. Ill dig a bit. May be worth, just fixing now within our code by adding the tags and removing the extension logic, then I may open a PR with the library and add yaml tags and see if I can get it approved.

Reply:
Fixed: f985fc627fbf00f75bb197b4d3b7b56006cfd654